pandas/tests/frame/test_reshape.py

```diff
@@ -560,16 +560,6 @@ def test_unstack_dtypes(self):
             assert left.shape == (3, 2)
             tm.assert_frame_equal(left, right)
 
-    def test_unstack_non_unique_index_names(self):
```

Comment:
what does this do now?

Reply:
It raises when you try to create the ``MultiIndex`` with duplicated name.